src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,59 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[];
+  selectedTabTitle: string;
+  selectedTabId: string;
+  selectedTabContent: string,
+  onSelectedTab: (id: string, title: string, content: string) => void,
+
+};
+
+export const Tabs:React.FC<Props> = ({
+  tabs,
+  selectedTabTitle,
+  selectedTabId,
+  selectedTabContent,
+  onSelectedTab,
+
+}) => (
+  <div className="section">
+    <h1 className="title">
+      {`Selected tab is ${selectedTabTitle}`}
+    </h1>
+
+    <div data-cy="TabsComponent">
+
```

Comment:
( I will comment in-order with the steps that will help you also pass the tests)

Keep the div with class section in the App component and move only the block with the data-cy="TabsComponent".